roles/cloud-vultr/tasks/prompts.yml

```diff
@@ -12,17 +12,21 @@
   set_fact:
     algo_vultr_config: "{{ vultr_config | default(_vultr_config.user_input) | default(lookup('env','VULTR_API_CONFIG'), true) }}"
 
+- name: Set the Vultr API Key as a fact
+  set_fact:
+    vultr_api_key: "{{ lookup('ansible.builtin.ini', 'key', section='default', file=algo_vultr_config) }}"
```

Comment:
When I reached the  "Enter the local path to your configuration INI file" prompt, I left it blank in hopes it would use `~/.vultr.ini` by default, but I got an error:
```
"msg": "An unhandled exception occurred while running the lookup plugin 'ansible.builtin.ini'. Error was a <class 'ansible.errors.AnsibleParserError'>, original message: an error occurred while trying to read the file '/Users/max/dev/algo/roles/cloud-vultr/': [Errno 21] Is a directory:
```

I'm guessing it didn't like the empty input?

However, after entering `~/.vultr.ini` directly, everything worked for me